src/app/workflow/view/view.service.ts

```diff
@@ -134,8 +166,11 @@ export class ViewService {
     if (selectedWorkflow.id === workflow.id) {
       this.workflowService.setWorkflow(workflow);
     }
+
+    const workflowName: string = workflow.workflowName == null ? workflow.repository : workflow.workflowName;
```

Comment:
Others may not agree, but my personal preference is to write this as `const workflowName: string = workflowName.workflowName || workflow.repository`. Feel free to ignore.

Reply:
I prefer yours.

Reply:
Just a reminder that it's not the same.  Although in this case, the coalesce (falsey check) probably works better.